Resources/Public/css/bootstrap.css

```diff
@@ -484,7 +484,7 @@ small,
 mark,
 .mark {
   background-color: #fcf8e3;
-  padding: .2em;
+  padding: 0.2em;
```

Comment:
Could you please add some description of why we need to add these changes? Also, keep in mind that `Resources/Public/css/bootstrap.css` is an auto-generated file, so all changes which were added manually will be ignored. 
Thanks in advance!

Reply:
Hi Dmytro,
File has been removed.
When generating this was randomly reformatted
Thanks for feedback!